.github/CODEOWNERS

```diff
@@ -1,12 +1,13 @@
 # Default owners for everything in the repo, unless a later match takes precedence.
-* @mbzomowski @RissyRan @allenwang28
+* @mbzomowski @RissyRan @allenwang28 @ManfeiBai
 
 dags/solutions_team/configs/tensorflow @chandrasekhard2 @ZhaoyueCheng @richardsliu
 dags/solutions_team/solutionsteam_tf* @chandrasekhard2 @ZhaoyueCheng @richardsliu
 
 dags/pytorch_xla @JackCaoG @vanbasten23 @zpcore @ManfeiBai
 dags/legacy_test/tests/pytorch @JackCaoG @vanbasten23 @zpcore @ManfeiBai
 
+dags/mantaray @ManfeiBai
```

Comment:
nit: add a space after the line?

Also, you may want to add an extra folk to unblock your PR review for this folder.

Reply:
Thanks, updated and added @tengyifei and @zpcore as the extra folk too